contents/blog/trademarks-blog-post.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-08-18
+title: Don’t Bother Securing Your Trademarks In The Beginning
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+Don’t bother securing your trademarks in the early days of your business. 
+
+In fact, if you’re yet to raise a pre-seed round, the money you’d spend on IP lawyers could probably be better spent elsewhere. In those first crucial months, you’ve got bigger problems to worry about - like finding product-market fit, hiring more (and better) people, improving your quality control, ramping up your marketing and [content](https://posthog.com/blog/running-content-at-posthog), and more. There’s also a good chance that your brand might evolve during that time - from name changes and logo revamps to brand fonts and colors. This makes trademarking a word mark or logo mark _that_ early a risky bet.
+
+## What is the different between a word mark and a logo mark?
+
+A word mark is typically just the name of the brand (as written in a specific font, if applicable). In our case, it’s the word ‘PostHog.’ A logo mark is typically pictorial - in our case, the hedgehog logo either alone or combined with a stylized word (see below). The word mark is the most versatile, so that should be your first trademark priority.
+
+[insert side by side comparison between the PH word mark and logo mark here]
```

Comment:
@lottiecoxon We'll need a quick extra graphic here as well - just the word mark and logo mark side by side.